packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -359,7 +359,18 @@ class TextSelectionOverlay {
   /// {@macro flutter.widgets.SelectionOverlay.showHandles}
   void showHandles() {
     _updateSelectionOverlay();
-    _selectionOverlay.showHandles();
+    // Desktop platforms don't have selection handles.
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.macOS:
+      case TargetPlatform.fuchsia:
+      case TargetPlatform.linux:
+      case TargetPlatform.windows:
+        break;
+      case TargetPlatform.android:
+      case TargetPlatform.iOS:
```

Comment:
what about iOS with blue tooth mouse?

Reply:
Good point, I should actually try that and see what happens.  I'm not sure how it handles showing the handles or not.

Reply:
With the new solution, mobile platforms with a mouse will behave the same as desktop platforms with a mouse (clicking and dragging with a mouse will select, not drag the cursor).